openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -118,22 +118,32 @@
 - control_key: AU-4
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: partial
+  implementation_status: not applicable
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
+        OpenShift does not control the storage capacity for the
+        OpenShift API audit logs when they are generated.
 
-        https://issues.redhat.com/browse/CMP-159
+        Either the OpenShift API audit logs need to be consumed
+        by an external SEIM and the storage capacity managed there
+        as per organization policy requirments or the operating system on
+        which OpenShift is running needs to have /var/log/audit storage
+        capacity configured to meet organization policy requirments.
```

Comment:
Where can documentation be found regarding how to plan appropriate storage capacity for OpenShift?

Also note that, apparently, OpenShift does not audit everything to /var/log/audit/* and has multiple log files on the system. Need to plan for the capacity of those too.

Unfortunately, AU-4(1), which requires off-loading audit records, isn't selected in many baselines. As such we'll need a control response akin to "If not offloading logs, here's how to plan for audit capacity. If offloading logs this control is deferred to external SIEM, and off-loading implementation guidance is provided in AU-4(1)"

Reply:
Further elaboration - when cluster logging is enabled, how do we ensure enough disk space was allocated for Elasticsearch? And on what partition(s) should it be allocated?

The OpenShift docs at https://docs.openshift.com/container-platform/4.3/logging/cluster-logging.html#cluster-logging-about-elasticsearch_cluster-logging redirects to https://www.elastic.co/guide/en/elasticsearch/guide/current/hardware.html, however the Elastic.co documentation is about *hardware*, not sizing.